### DISCUSSION TITLE
wind_gusts_10m_max returns values for only 5 days in Icon EU

### DISCUSSION THREAD

**Discussion by magheralex** [open-meteo/open-meteo#649] [General]
Hi there,
Icon EU provides data for seven days. However, I noticed that wind_gusts_10m_max returns values for the first five days.
 Is there a bug?

"daily":{"time":["2024-02-11","2024-02-12","2024-02-13","2024-02-14","2024-02-15","2024-02-16","2024-02-17"],"wind_gusts_10m_max":[31.7,33.8,27.4,18.7,19.4,null,null]}}

eg. https://api.open-meteo.com/v1/forecast?latitude=44.4323&longitude=26.1063&hourly=temperature_2m&daily=wind_gusts_10m_max&timezone=auto&models=icon_eu

**Comment by patrick-zippenfenig** [comment]
ICON EU only offers 5 days of forecast. One the global ICON domain has 7.5 days of forecast. If you select `icon_seamless` the API will automatically combine ICON-D2, EU and Global.

**Reply by magheralex** [reply]
Hi @patrick-zippenfenig ,
Thanks for your answer. Is there any option to automatically combine ICON-D2, ICON-EU, ECMWF_IFS025, and GFS Global? or ICON-EU, ECMWF_IFS025, and GFS-Global?

**Reply by patrick-zippenfenig** [reply]
No, it is not possible to arbitrarily combine models. `icon_seamless` is basically [hardcoded](https://github.com/open-meteo/open-meteo/blob/main/Sources/App/Controllers/ForecastapiController.swift#L327). It is feasible to make this more dynamic, but would requite some larger refactoring (Not to mention the interface / documentation would be quite complex.)
